src/absil/il.fs

```diff
@@ -13,22 +13,17 @@ open Microsoft.FSharp.Compiler.AbstractIL
 open Microsoft.FSharp.Compiler.AbstractIL.Internal
 open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library
 open Microsoft.FSharp.Compiler.AbstractIL.Diagnostics
-open System
 open System.Collections
 open System.Collections.Generic
 open System.Collections.Concurrent
  
 let logging = false 
 
-let runningOnWindows =
-    match System.Environment.OSVersion.Platform with
-    | PlatformID.Win32NT | PlatformID.Win32S | PlatformID.Win32Windows | PlatformID.WinCE -> true
-    | _  -> false
-
+let runningOnMono = 
```

Comment:
I don't yet understand why these changes were not integrated across from Microsoft\visualfsharp in the latest integration.  I will look into that.  You shouldn't need to be bringing them across - they should just be there